images/install_greenplum.sh

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
I'd put a shebang here (e.g. `#!/usr/bin/env bash`)

Reply:
Should we put a license header here, too, as in some of the other files?

Reply:
Done.